src/System.Private.CoreLib/src/Internal/Diagnostics/Tracing/FrameworkEventSourceSupport.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+
+namespace Internal.Diagnostics.Tracing
+{
+    //
+    // An internal contract that exposes just enough async debugger support needed by the AsTask() extension methods in the WindowsRuntimeSystemExtensions class.
```

Comment:
This comment does not match the implementation.

Reply:
Fixed